docs/data/data-grid/column-definition/SparklineColumn.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from 'react';
+import {
+  DataGrid,
+  GridColDef,
+  GridRowsProp,
+  GridColTypeDef,
+  GridRenderCellParams,
+  GRID_STRING_COL_DEF,
+} from '@mui/x-data-grid';
+import { LineChart } from '@mui/x-charts/LineChart';
+
+interface SparklineProps {
+  data: number[];
+  width: number;
+}
+
+function Sparkline(props: SparklineProps) {
+  const { data, width } = props;
+
+  const xAxisData = React.useMemo(
+    () => Array.from({ length: data.length }).map((_, index) => index),
+    [data],
+  );
+
+  return (
+    <LineChart
+      margin={{ left: 6, right: 6, top: 4, bottom: 4 }}
+      yAxis={[{ hidden: true }]}
+      xAxis={[{ hidden: true, data: xAxisData }]}
```

Comment:
If I'm correct (a bit lost between what is already committed and what is in open PR)

Your `xAxisData` is already the default `xAxis.data` provided for `LineCharts`

To hide axis, you can use `null` (here is the [docs example](https://mui.com/x/react-charts/axis/#position))

```suggestion
      leftAxis={null}
      bottomAxis={null}
```

Reply:
It would be nice to add this to the doc, I did not find it myself :+1:

Reply:
where would you add it?

Reply:
So splitting the [Position](https://deploy-preview-9228--material-ui-x.netlify.app/x/react-charts/axis/#position) section in two: "hide the axis" and "place the axis"